### DISCUSSION TITLE
My orca Slicer, closes itself

### DISCUSSION THREAD

**Discussion by Migu7e** [OrcaSlicer/OrcaSlicer#7403] [Dev]
I really crazy, because I work for 5 minutes and it closes anytime, I lose my all my progress, I have the 2.2.0 version, help me please. I work in my PC:
Asus Z790-E
i7-13000
32 gigas RAM
RTX 4070 super
SSD M.2 high velocity
Aourus waterforce 360

**Comment by Migu7e** [comment]
I really angry for this bug or crash or problem, please help me

**Comment by coopa09** [comment]
I have the problem that during the dynamic test, it simply stops at different points, supposedly due to an issue with the nozzle temperature. When I use Bambu Studio, I don’t have any problems with it.

**Comment by ButtlerDrownPF** [comment]
[Orca Slicer Latest Version Download ](https://theorcaslicer.com/)is really good slicing software that converts 3D models into G-Code. It also enhances printing performance and makes it smooth. It also gives you the variety of customization options for printing.
